app/App/Panel/Main/Account/Requests/TransactionRequest/index.js

```diff
@@ -419,7 +415,23 @@ class TransactionRequest extends React.Component {
         )}
         {!notice ? (
           <div className='requestApprove'>
-            {req.recentFeeUpdate ? <div className='requestApproveBlock cardShow'>{'transaction fee updated'}</div> : null}
+            {req.automaticFeeUpdateNotice ? (
+              <div className='requestApproveFeeBlock cardShow'>
+                <div className='requestApproveFeeButton requestApproveFeeReject' onClick={() => {
+                  const { previousFee } = req.automaticFeeUpdateNotice
+                  if (previousFee.type === '0x2') {
```

Comment:
I'm not following the logic here. if the previousFee was an eip-1559 tx then we're setting the base/priority fees? shouldn't it be the opposite where we're removing them because we're converting _away_ from an eip-1559 tx?

Reply:
`previousFee` is unrelated to the london > legacy, it allows users to accept/rollback automatic fee updates

Reply:
gotcha. maybe something like `previousTxFees`?